cli/localci/cli/run.py

```
@@ -118,20 +133,25 @@ def run(
     timeout: int | None,
     dry_run: bool,
     no_cache: bool,
+    cache_dir: Path | None,
     rebuild_image: bool,
-    keep_containers: bool,
+    keep_containers: bool | None,
     interactive: bool,
     verbose: bool,
```

Ruff (0.15.4)

[warning] Unused function argument: `verbose`

(ARG001)